plantswap/components/Form/Form.js

```diff
@@ -0,0 +1,116 @@
+'use client'
+
+import React from 'react';
+
+import styles from "./form.module.scss";
+
+const Form = ({ data }) => {
```

Comment:
Implement the dictionary.
{data, dictionary}

Scroll completely down to see how to add dictionary